.vscode/settings.json

```diff
@@ -20,5 +20,6 @@
     "**/dist": true,
     "node_modules/**": true,
     "packages/node_modules/**": false
-  }
+  },
+  "files.eol": "\n"
```

Comment:
Make sure this is also configured in `.editorconfig`. That's really the right way to do this sort of thing (though also support vscode directly doesn't hurt. It's kinda unfortunate vscode doesn't just build in editorconfig.